test/e2e/web_console_plugin_test.go

```diff
@@ -0,0 +1,153 @@
+package e2e
+
+import (
+	"crypto/tls"
+	"fmt"
+	testconfig "github.com/codeready-toolchain/toolchain-common/pkg/test/config"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+	v1 "github.com/openshift/api/route/v1"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	wait2 "k8s.io/apimachinery/pkg/util/wait"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+)
+
+type webConsolePluginTest struct {
+	suite.Suite
+	wait.Awaitilities
+}
+
+func TestRunWebConsolePluginTest(t *testing.T) {
+	suite.Run(t, &webConsolePluginTest{})
+}
+
+func (s *webConsolePluginTest) SetupSuite() {
+	s.Awaitilities = WaitForDeployments(s.T())
+}
+
+func (s *webConsolePluginTest) TearDownTest() {
+	hostAwait := s.Host()
+	memberAwait := s.Member1()
+	memberAwait2 := s.Member2()
+	hostAwait.Clean(s.T())
+	memberAwait.Clean(s.T())
+	memberAwait2.Clean(s.T())
```

Comment:
I'm not sure if we need to call the clean when the clean functions are already registered as part of the creation step